kart/tile/importer.py

```diff
@@ -541,3 +608,22 @@ def missing_parameter(self, param_name):
 
     def sidecar_files(self, source):
         return []
+
+    MAX_WORKERS = 64
+
+    def check_num_workers(self, num_workers):
+        if num_workers is None:
+            num_workers = self.get_default_num_workers()
+        if num_workers < 1:
+            num_workers = 1
+        elif num_workers > self.MAX_WORKERS:
+            raise click.UsageError(
+                f"Importing with more than {self.MAX_WORKERS} workers is not supported"
```

Comment:
why? this was here for the old v2 tabular import code because each process handled one dest subtree, but it doesn't seem like it makes sense for tile-based imports

Reply:
Removed arbitrary MAX_WORKERS limit